### DISCUSSION TITLE
Installing Spiderplan & Aries

### DISCUSSION THREAD

**Discussion by Yomaster10** [aiplan4eu/unified-planning#273] [Q&A] [answered]
How do I go about installing the Spiderplan and Aries engines (Windows 11 or Ubuntu 18/20)?

**Comment by Framba-Luca** [comment, marked as answer]
Currently, the integration of Spiderplan and Aeries is under development.

I don't know whether they are usable or not, but the installer `pip install unified-planning[aeries,spiderplan]` doesn't work.

@arbimo for Aeries and @uwe-koeckemann for Spiderplan can give you more accurate answers!
